app/client/src/components/editorComponents/CodeEditor/index.tsx

```diff
@@ -386,6 +393,9 @@ class CodeEditor extends Component<Props, State> {
           entityInformation.entityType = entityType;
         }
       }
+      if (isActionEntity(entity)) entityInformation.entityId = entity.actionId;
+      if (isWidgetEntity(entity)) entityInformation.entityId = entity.widgetId;
```

Comment:
nitpick: does this work too?
```suggestion
     entityInformation.entityId = isActionEntity(entity) ? entity.actionId : entity.widgetId;
```